src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --primary-color: #00acdc;
+  --main-text-color: #060b35;
+  --text-secondary-color: #616070;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.product__name {
+  font-size: 12px;
+  margin: 0 0 4px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--main-text-color);
+}
+
+.product__code {
+  margin: 0 0 16px;
+  font-size: 10px;
+  color: #616070;
+}
+
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+  line-height: 18px;
+}
+
+.product__price-label {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.product__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-text-color);
+}
+
+.product__cta-link {
+  height: 40px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
```

Comment:
if you want to reset margins, do it in one place, along with the body and the p tag. but here there's nothing to reset. links dont have default margins

Reply:
Thank you have corrected